assets/prometheus-k8s/service-monitor-kubelet.yaml

```diff
@@ -7,54 +7,55 @@ metadata:
   namespace: openshift-monitoring
 spec:
   endpoints:
-  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
-    honorLabels: true
-    interval: 30s
-    port: https-metrics
-    relabelings:
-    - sourceLabels:
-      - __metrics_path__
-      targetLabel: metrics_path
-    scheme: https
-    tlsConfig:
-      caFile: /etc/prometheus/configmaps/kubelet-serving-ca-bundle/ca-bundle.crt
-      insecureSkipVerify: false
```

Comment:
is this now left out by purpose?

Reply:
It's git being silly, it just moved to the end of the file, because we needed to apply the relabelling to the other endpoint.